reduce_points_factor: 4
desired_freq: 4.0 # in Hz